src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/InstancesController.cs

```diff
@@ -173,7 +195,24 @@ public class InstancesController : ControllerBase
                 continuationToken = HttpUtility.UrlDecode(continuationToken);
             }
 
-            Dictionary<string, StringValues> queryParams = QueryHelpers.ParseQuery(Request.QueryString.Value);
+            Dictionary<string, StringValues> queryParams = new Dictionary<string, StringValues>()
+            {
+                { "org", org },
+                { "appId", appId },
+                { "process.currentTask", currentTaskId },
+                { "process.isComplete", processIsComplete?.ToString() },
+                { "process.endEvent", processEndEvent },
+                { "process.ended", processEnded },
+                { "instanceOwner.partyId", instanceOwnerPartyId?.ToString() },
+                { "lastChanged", lastChanged },
+                { "created", created },
+                { "visibleAfter", visibleAfter },
+                { "dueBefore", dueBefore },
+                { "excludeConfirmedBy", excludeConfirmedBy },
+                { "status.isSoftDeleted", isSoftDeleted ? "true" : null },
+                { "status.isHardDeleted", isHardDeleted ? "true" : null },
+                { "status.isArchived", isArchived ? "true" : null },
+            }.Where(el => el.Value != StringValues.Empty).ToDictionary(x => x.Key, x => x.Value);
```

Comment:
Ooops! I just noticed that this approach will break a query on the form `dueBefore=gt:2019-02&dueBefore=lt:2019-03-01`. I've never seen that pattern before.

Maybe it can be fixed by changing the type signature for the dateTime query elements to `StringValues`, and maybe the API docs will match the API better too. The very minimum would be integration tests that ensures that date time range queries work.

Reply:
`instanceOwner.PartyId` is another example where we currently want to support multiple values through repeated use of the same parameter. 

For now I would like to see this particular change reverted.